apps/public-docsite-v9/package.json

```diff
@@ -33,6 +33,7 @@
     "@fluentui/react-icons-northstar": "*",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31705&commitId=4e944ae66179b748be1e615dc513498bf0ef0b07&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|5| Changed|
|Avatar Converged.basic - RTL.normal.chromium.png|0| Removed|
|Avatar Converged.size+inactive+badge.normal.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>